Test/Tests/RealtimeClientConnectionTests.swift

```diff
@@ -3838,7 +3838,7 @@ class RealtimeClientConnectionTests: XCTestCase {
     func test__086__Connection__Host_Fallback__failing_connections_with_custom_endpoint_should_result_in_an_error_immediately() {
         let test = Test()
         let options = ARTClientOptions(key: "xxxx:xxxx")
-        options.environment = "test" // do not use the default endpoint
```

Comment:
Why are these changes separate from 88611490120f3ad93565deca454903bf4a1ffbcb? Aren't they fixing test failures introduced by a change in that commit?

Reply:
e.g.

![image](https://github.com/user-attachments/assets/3cbcbce9-b39f-4894-b70d-02f2bafb01a0)

vs

![image](https://github.com/user-attachments/assets/348ed3fa-d8f7-4c65-8b69-1d0d914572ac)

Reply:
https://github.com/ably/ably-cocoa/pull/2017/commits/2a9b845dad248acaa9cfc2584bd3a52481749f63